mm2src/coins/utxo/utxo_common.rs

```diff
@@ -2635,7 +2635,7 @@ pub fn get_trade_fee<T: UtxoCommonOps>(coin: T) -> Box<dyn Future<Item = TradeFe
 ///
 /// To sum up, `get_sender_trade_fee(TradePreimageValue::Exact(9000)) > get_sender_trade_fee(TradePreimageValue::Exact(10000))`.
 /// So we should always return a fee as if a transaction includes the change output.
-pub async fn preimage_trade_fee_required_to_send_outputs<T>(
+pub async fn preimage_trade_fee_required_to_send_outputs<T: MarketCoinOps>(
```

Comment:
First, we already have trait bounds for the generic `T` type:
```rust
where
    T: UtxoCommonOps + GetUtxoListOps,
```
It had to be there.

Secondly, `T: MarketCoinOps` is not required due to the next comment